controllers/ControllerAdmin.php

```diff
@@ -209,37 +220,37 @@ public function avatar(){
 		$key = (new Session)->getKey('user');
 
 		if($_FILES['avatar']['error'] > 0){
-			$error =true;
-			echo $this->_twig->render('profile.twig', ['error' => $error,
-			                                           'server' => $this->_server,
-			                                           'logged'=> TRUE,
-			                                           'user'=> $key['pseudo'],
-			                                           'value'=> $key['email'],
-			                                           'avatar'=> $key['avatar']]);
+			echo $this->getTwig()->render('profile.twig', ['error' => TRUE,
+			                                               'server' => $this->getServer(),
+			                                               'logged'=> TRUE,
+			                                               'user'=> $key['pseudo'],
+			                                               'value'=> $key['email'],
+			                                               'admin' => $key['admin'],
+			                                               'avatar'=> $key['avatar']]);
 			exit();
 		}
 
 		if ($fileSize > $maxSize){
-			$size = true;
-			echo $this->_twig->render('profile.twig', ['size' => $size,
-			                                           'server' => $this->_server,
-			                                           'logged'=> TRUE,
-			                                           'user'=> $key['pseudo'],
-			                                           'value'=> $key['email'],
-			                                           'avatar'=> $key['avatar']]);
+			echo $this->getTwig()->render('profile.twig', ['size' => TRUE,
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)